src/ot_orchestration/dags/config/unified_pipeline.yaml

```diff
@@ -100,3 +100,4 @@ etl_steps:
   - name: search
     depends_on:
       - etl_association
+      - genetics_variant_index
```

Comment:
Might out of scope, but the `evidence` step depends on `genetics_l2g_evidence` step.